From 557be66b9451f83a565bf22781a65664e76e2a39 Mon Sep 17 00:00:00 2001
From: Wendy Liang <wendy.liang@xilinx.com>
Date: Mon, 5 Aug 2019 23:45:41 -0700
Subject: [PATCH 0651/1851] dt-bindings: soc: xilinx: Correct ai_engine reg
 property

commit fcaa3e20ee0dc10f6ce2f12c4f5f99bce810fd41 from
https://github.com/Xilinx/linux-xlnx.git

Correct the reg property of the ai_engine device node.

Signed-off-by: Wendy Liang <wendy.liang@xilinx.com>
Signed-off-by: Michal Simek <michal.simek@xilinx.com>
State: pending
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 .../devicetree/bindings/soc/xilinx/xlnx,ai_engine.txt         | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/Documentation/devicetree/bindings/soc/xilinx/xlnx,ai_engine.txt b/Documentation/devicetree/bindings/soc/xilinx/xlnx,ai_engine.txt
index 04244a6bc2da..b7643a1380d7 100644
--- a/Documentation/devicetree/bindings/soc/xilinx/xlnx,ai_engine.txt
+++ b/Documentation/devicetree/bindings/soc/xilinx/xlnx,ai_engine.txt
@@ -17,9 +17,9 @@ Required properties:
 
 Example:
 
-	ai_engine@80000000 {
+	ai_engine@20000000000 {
 		compatible = "xlnx,ai_engine";
-		reg = <0x0 0x80000000 0x0 0x20000000>;
+		reg = <0x200 0x0 0x1 0x0>;
 		interrupt-parent = <&gic>;
 		interrupts = <0x0 0x94 0x1>,
 			     <0x0 0x95 0x1>,
-- 
2.31.1

